src/extensions.msal/Microsoft.Identity.Client.Extensions.Msal/Microsoft.Identity.Client.Extensions.Msal.csproj

```diff
@@ -19,17 +20,22 @@
 
   <PropertyGroup>
     <!-- The MSAL.snk has both private and public keys -->
-    <DelaySign>false</DelaySign>
+    <DelaySign>false</DelaySign>    
   </PropertyGroup>
 
   <PropertyGroup>
 
-    <!--This should be passed from the VSTS build-->
-    <ClientSemVer Condition="'$(MsalClientSemVer)' == ''">4.0.0-localbuild</ClientSemVer>
     <!-- For now, the Extensions.Msal libraries are ALWAYS in preview mode -->
-    <MsalClientSemVer Condition="'$(MsalClientSemVer)' != ''">$(MsalClientSemVer)-preview</MsalClientSemVer>
-    <!--This will generate AssemblyVersion, AssemblyFileVersion and AssemblyInformationVersion-->
-    <Version>$(MsalClientSemVer)</Version>
+    <DesktopExtensionVersion Condition="'$(ExtensionUsesMsalFromNuGetVersion)' == '' and '$(MsalClientSemVer)' != ''">
+      $(MsalClientSemVer)-preview
+    </DesktopExtensionVersion>
+    <DesktopExtensionVersion  Condition="'$(ExtensionUsesMsalFromNuGetVersion)' != ''" >
+      $(ExtensionUsesMsalFromNuGetVersion)-preview
+    </DesktopExtensionVersion>
+    <DesktopExtensionVersion Condition="$(DesktopExtensionVersion) == ''">4.5.0-internal</DesktopExtensionVersion>
```

Comment:
Is it something that is hardcoded?

Reply:
Yes, but I will remove this as there are so many defaults now ...